app/liquid_tags/unified_embed/tag.rb

```diff
@@ -41,7 +44,7 @@ def self.new(tag_name, link, parse_context)
       # Why the __send__?  Because a LiquidTagBase class "privatizes"
       # the `.new` method.  And we want to instantiate the specific
       # liquid tag for the given link.
-      klass.__send__(:new, tag_name, validated_link, parse_context)
+      klass.__send__(:new, tag_name, stripped_link, parse_context)
```

Comment:
Can you help me understand the reason this changed from `validated_link` to `stripped_link`?

Reply:
@jeremyf In the relatively rare case that we are embedding a Github Repo URL with the `noreadme` option, passing `validated_link` will forward only the URL (without the option) to the GithubTag class.

In my mind, we can still validate the link without actually passing the link used for the validation through, since the logic in this file raises appropriate errors if the link is invalid. After validating the `stripped_link`, we can continue using the `stripped_link`

I'm open to more elegant solutions. 🙏🏾

Reply:
This explanation makes sense to me, and with this explanation we're in "unblocked" territory, but I'm trying to think through my understanding of the variable names.

- `stripped link` is closest to the original it has URL and possible options, should we rename that?
- `actual_link` is URL without options
- `validated_link` is the same as `actual_link`

So what I think we could do is rename the `validate_link` method to a `validate_link!` (like we just want to know that the link works).  We don't need to capture the `validated_link` variable as that's what we passed to `validate_link`.

Does that align with your understanding?

Reply:
Makes sense! I think for `stripped_link`, we could call it `stripped_input`

Reply:
Yes to something simple and general. (Update the parameter documentation to say what it might look like)

Reply:
Sorry for the miscommunication, I was going from memory.  I was assuming the input parameter (for the method) would be input…it was jarring to see link become something generalized.

But I see the change.  It's definitely non-blocking, and I'm going to approve yet see how you want to proceed.